conan/tools/microsoft/msbuilddeps.py

```diff
@@ -109,11 +109,12 @@ def __init__(self, conanfile):
                          'x86_64': 'x64'}.get(str(conanfile.settings.arch))
         # ca_exclude section
         self.exclude_code_analysis = None
-        ca_exclude = self._conanfile.conf["tools.microsoft.msbuilddeps:exclude_code_analysis"]
+        ca_exclude = self._conanfile.conf.get("tools.microsoft.msbuilddeps:exclude_code_analysis")
         if ca_exclude is not None:
             # TODO: Accept single strings, not lists
-            self.exclude_code_analysis = eval(ca_exclude)
-            if not isinstance(self.exclude_code_analysis, list):
+            if isinstance(ca_exclude, list):
```

Comment:
In this case, it would be enough with that `conf.get("tools.microsoft.msbuilddeps:exclude_code_analysis")`, without having to convert it because the user knows that this field is a `list`:

```
[conf]
tools.microsoft.msbuilddeps:exclude_code_analysis = ['pkga*']
```
So, this refactor brings us:

```python
conf.get("tools.microsoft.msbuilddeps:exclude_code_analysis") == ["pkga*"]
```

Reply:
Yes, ``check_type`` is even better 👍